setup.py

```diff
@@ -96,7 +96,10 @@ def run(self):
 # Extra dependencies for remote integrations
 
 gs = ["gcsfs==2021.7.0"]
-gdrive = ["pydrive2>=1.8.1", "six >= 1.13.0"]
+gdrive = [
+    "pydrive2[fsspec] @ git+https://github.com/iterative/pydrive2@fsspec",
+    "six >= 1.13.0",
+]
```

Comment:
For the record: 1.9.0 is out and we no longer need `six` here
```suggestion
gdrive = ["pydrive2[fsspec]>=1.9.0"]
```